### ISSUE TITLE
Announcement: Maintenance Mode 🔧

### ISSUE THREAD

**Issue by seansfkelley** [seansfkelley/nas-download-manager#227]
tl;dr: NAS Download Manager is in maintenance mode. No new features are planned, and only the highest priority bug fixes, translations, or a fix to #7 will be addressed.

NAS Download Manager started as a passion project to fill a feature gap and improve upon the existing browser extensions. I published it publicly primarily to give back to the community, and have been pleasantly surprised at how many others found it useful. I benefited from the community as well, receiving bug fixes, feature improvements and translations, the latter of which I could never have done on my own. A special thank you goes out to my Firefox users, who were the first audience and contributed the most, by any measure, to the success of this extension.

The project isn't dead, but it's important to recognize that it can't be everything to everyone and at some point I want it to be considered done. I've always maintained this extension for free and on my own time, of which I have had less of in recent years. The feature set has stabilized and most new feature requests are out of the scope I intend for the extension; indeed, it has served my own needs (and, I suppose, most of my users') just fine for years. Now is as good a time as any.

I plan to keep the source available indefinitely, but I do not plan to transfer ownership to anyone else. I pride myself on having built a successful community project by staying focused, responding to user feedback directly, and without resorting to privacy-invading analytics. I have seen too many projects turn sour when their original creators leave, and would much prefer NAS Download Manager to become slowly irrelevant than risk its corruption. Anyone will remain free to fork it and release their own version under the terms of the MIT license, as has always been indicated in `package.json`.

As indicated in the tl;dr, the following kinds of changes will still be accepted:

- high priority bug fixes: security issues, DSM compatibility issues, browser compatibility issues, or large-scale breakages
- translations: fixes to existing translations as well as new translations
- a ready-to-merge fix to [#7](https://github.com/seansfkelley/nas-download-manager/issues/7): I never had time to correctly resolve the most-requested feature, and I was always disappointed in its absence

Adios.
